public/app/features/explore/state/time.test.ts

```diff
@@ -39,7 +39,7 @@ describe('Explore item reducer', () => {
       });
       await dispatch(updateTime({ exploreId: ExploreId.left }));
       expect(mockTimeSrv.init).toBeCalled();
-      expect(mockTemplateSrv.updateTimeRange).toBeCalledWith(MOCK_TIME_RANGE);
+      expect(mockTemplateSrv.updateTimeRange).toBeCalledWith(MOCK_TIME_RANGE, undefined);
```

Comment:
FYI, with `defaultTimeZone` from `@grafana/schema`, my first rewrite of this test was as follows. It failed.

```js
expect(mockTemplateSrv.updateTimeRange).toBeCalledWith(MOCK_TIME_RANGE, defaultTimeZone);
```